example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+    '13.338131951560397,50.73323436136428,' +
+    '12.96683678554325,50.48407644306917,' +
+    '12.240111118222671,50.26633779560723,' +
+    '12.415190870827473,49.96912079528062,' +
+    '12.521024204161336,49.54741526956275,' +
+    '13.031328973043514,49.30706818297324,' +
+    '13.595945672264577,48.877171942737164,' +
+    '13.243357374737116,48.41611481382903,' +
+    '12.884102817443873,48.28914581968786,' +
+    '13.025851271220517,47.63758352313595,' +
+    '12.932626987366064,47.467645575544,' +
+    '12.620759718484521,47.672387600284424,' +
+    '12.141357456112871,47.70308340106578,' +
+    '11.426414015354851,47.52376618101306,' +
+    '10.544504021861597,47.5663992376538,' +
+    '10.402083774465325,47.30248769793916,' +
+    '9.896068149463188,47.580196845075704,' +
+    '9.594226108446376,47.5250580918202,' +
+    '8.522611932009795,47.83082754169135,' +
+    '8.317301466514095,47.61357982033627,' +
+    '7.466759067422288,47.62058197691192,' +
+    '7.593676385131062,48.33301911070373,' +
+    '8.099278598674855,49.01778351500343,' +
+    '6.658229607783709,49.20195831969164,' +
+    '6.186320428094177,49.463802802114515,' +
+    '6.242751092156993,49.90222565367873,' +
+    '6.043073357781111,50.128051662794235,' +
+    '6.156658155958779,50.80372101501058,' +
+    '5.988658074577813,51.851615709025054,' +
+    '6.589396599970826,51.852029120483394,' +
+    '6.842869500362383,52.22844025329755,' +
+    '7.092053256873896,53.14404328064489,' +
+    '6.905139601274129,53.48216217713064,' +
+    '7.100424838905268,53.69393219666267,' +
+    '7.936239454793962,53.74829580343379,' +
+    '8.121706170289485,53.52779246684429,' +
+    '8.800734490604668,54.020785630908904,' +
+    '8.572117954145368,54.39564647075405,' +
+    '8.526229282270208,54.96274363872516,' +
+    '9.282048780971136,54.83086538351631,' +
+    '9.921906365609232,54.983104153048025';
+
+// API URL
+const API_URL = 'https://demo.emissions-api.org/api/v1/geo.json?' +
+    GERMANY + '&begin=2019-02-04&end=2019-02-06';
+
+// Helper function to calculate the average value
+function average(numbers) {
+    let total = 0;
+    for (let i = 0; i < numbers.length; i += 1) {
+        total += numbers[i];
+    }
+    return total / numbers.length;
+}
+
+new deck.DeckGL({
+    container: 'container',
+    mapboxApiAccessToken: '<your mapbox token>',
+    mapStyle: "mapbox://styles/mapbox/dark-v9",
+    longitude: 10,
+    latitude: 50,
+    zoom: 5,
+    pitch: 50,
+    layers: [
+        new deck.HexagonLayer({
+            extruded: true,
+            radius: 5000,
+            data: API_URL,
+            dataTransform: d => d.features,
+            elevationScale: 100,
+            getColorValue: points => average(points.map(d => d.properties.carbonmonoxide)),
+            getElevationValue: points => average(points.map(d => d.properties.carbonmonoxide)),
+            getPosition: d => d.geometry.coordinates,
+        }),
+    ]
+});
+```
+
+That's it!
```

Comment:
Maybe instead of this last phrase, you could link to a file that contains the example code and encourage playing around with it. Something along the lines of:
'Now go ahead and fork the example code `[here](some/git/example)` to start having fun with the code yourself!'

Reply:
I like the idea but I hate it if people use `here` links. So, maybe:

```
Now go ahead and [fork the example](some/git/example) to start having fun with the code yourself.
```

Reply:
Good idea, maybe we should add that, if/when we have a repository with example :smile:

Reply:
I will set up an example repository and we can than change this last phrase. For now I would resolve this conversation in order to not delay merging until thursday.